hw/bitstream/vivado/BUILD

```diff
@@ -103,6 +107,10 @@ fusesoc_build(
         "bitstream": ["synth-vivado/lowrisc_systems_chip_earlgrey_cw310_hyperdebug_0.1.bit"],
         "rom_mmi": ["synth-vivado/rom.mmi"],
         "otp_mmi": ["synth-vivado/otp.mmi"],
+        "logs": [
+            "synth-vivado/lowrisc_systems_chip_earlgrey_cw310_hyperdebug_0.1.runs/synth_1/runme.log",
```

Comment:
Would it make sense to output the whole run directory?

Reply:
we could potentially make the output_group parameter more general so it's told if it's a file or a directory (or even a directory that you want to tar). But unless there is a specific usage for it, I don't know if there is much point in adding support for it with no user.

Reply:
I've added this, seems to work fine